recipes/rosbuild/build.sh

```diff
@@ -0,0 +1,30 @@
+mkdir build
+cd build
+
+# turn this into a catkin workspace by adding the catkin token if it doesn't
+# exist already
+if [ ! -f $PREFIX/.catkin ]; then
+    touch $PREFIX/.catkin
+fi
+
+# necessary for correctly linking SIP files (from python_qt_bindings)
+export LINK=$CXX
+
+export ROS_PYTHON_VERSION=`${PYTHON} -c "import sys; print('%i.%i' % (sys.version_info[0:2]))"`
```

Comment:
Here it can be just
```suggestion
export ROS_PYTHON_VERSION=${PY_VER}
```

Reply:
Hmm, that's useful. One problem is that these recipes are auto-generated... I'll change this in my script that generates these recipes, and the next generation will be improved.